deployments/README.md

```diff
@@ -9,4 +9,24 @@ but the files contained here are useful for documentation and manual testing.
 * `flp-daemonset-cap.yml`, same as `flp-daemonset.yml`, but assigning individual capabilities instead
   of deploying a fully-privileged container.
 * `flp-service.yml`, shows how to deploy/configure the Agent when Flowlogs Pipeline is deployed
-  as a service, explicitly setting the host configuration as the service name.
\ No newline at end of file
+  as a service, explicitly setting the host configuration as the service name.
+
+For manual testing, create the netobserv namespace and first deploy loki.
+
+'''
+kubectl create namespace netobserv
+curl -S -L https://raw.githubusercontent.com/netobserv/documents/main/examples/zero-click-loki/1-storage.yaml | kubectl create -n netobserv -f - 
+curl -S -L https://raw.githubusercontent.com/netobserv/documents/main/examples/zero-click-loki/2-loki.yaml       | kubectl create -n netobserv -f - 
+'''
```

Comment:
Do we really want to keep loki in the examples ?

We could simply have stdout here and then the users can dig into different export capabilities.

Reply:
I don't have a strong preference. The existing example assumed loki, so I kept it that way. I simply added the instructions on how to bring up loki so that it will work.